scripts/k8s-storage/driver-rbd.yaml

```diff
@@ -56,6 +56,9 @@ DriverInfo:
     # Support for volume expansion in nodes
     nodeExpansion: true
 
+    # supports offline volume expansion
+    offlineExpansion: false
```

Comment:
@nixpanic same need to be done for cephfs and nfs driver as well?

Reply:
no, CephFS and NFS can be expanded while the volume is not mounted.